### ISSUE TITLE
Document the `playground` Cargo metadata

### ISSUE THREAD

**Issue by dns2utf8** [rust-lang/rust-playground#192]
Hey team

I made a test with the shiny new tool: https://play.rust-lang.org/?gist=118112c0ca3014d3683dfd6db7fa99ef&version=stable

Can you enable some the features for the crate?

**Comment by shepmaster** [comment]
> Can you enable some the features for the crate?

No, we do not wish to have *anything* to do with how any particular crate is exposed. Instead, you can add the `playground` feature to the libraries Cargo.toml. Here's [how Serde does it](https://github.com/serde-rs/serde/blob/26098ed877e18ce093549f9744424b3b4f83bc59/serde/Cargo.toml#L65-L68).

**Comment by shepmaster** [comment]
I'm going to repurpose this issue for adding documentation about this feature.

**Comment by shepmaster** [comment]
From #326

> This is based on the custom metadata configurable for docs.rs -- see https://docs.rs/about. Ours looks like:
>
> 
> ```rust 
> # Cargo.toml
> [package]
> name = "test"
> 
> [package.metadata.playground]
> default-features = true
> features = ["std", "extra-traits"]
> all-features = false
> ```
